packages/appium/docs/en/guides/caps.md

```diff
@@ -2,9 +2,150 @@
+
+## Appium capabilities
+
+Appium understands these browser-focused capabilities, but introduces a number of additional
+capabilities you are free to use to start Appium sessions. According to the WebDriver spec, any
+non-standard "extension capabilities" must include a namespace prefix (signifying the vendor
+introducing the capability), separated from the capability name by a `:`. Appium's vendor prefix is
+`appium:`, and so any Appium-specific capabilities must include this prefix. Depending on which
+client you are using, the prefix may be added automatically, or you might need to ensure they are
+included in capability names yourself (prefixes are added, for example, in the Appium Python
+client, but not the case in WebdriverIO).
+
+Here is a list of all the globally-recognized Appium capabilities. Notes:
+
+- Individual drivers and plugins can support other capabilities, so refer to their documentation for lists of extension-specific
+capability names.
+- Some drivers might not implement support for all of these capabilities
+- A few Appium extension capabilities are required to start any Appium session; these are noted below as "Required"
+
+| Capability&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;                            | Type      | Required? | Description                                                                                                                                                     |
+|---------------------------------------|-----------|-----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `platformName`                        | `string`  | Required  | The type of platform hosting the app or browser                                                                                                                 |
+| `appium:automationName`               | `string`  | Required  | The name of the Appium driver to use                                                                                                                            |
+| `browserName`                         | `string`  | Optional  | The name of the browser to launch and automate, if the driver supports web browsers as a special case                                                           |
+| `appium:app`                          | `string`  | Optional  | The path to an installable application                                                                                                                          |
+| `appium:deviceName`                   | `string`  | Optional  | The name of a particular device to automate, e.g., `iPhone 14`                                                                                                  |
```

Comment:
this might be confusing. For example UIA2 and XCUITest drivers ignore this capability while selecting a real device and respect the `udid` one.

Reply:
curious; why is that?

Reply:
🤷 
It was always working like that. Most likely, some historical reasons...
The only situation where deviceName is actually used is iOS simulator selection

Reply:
yeah, support for `deviceName` could theoretically be broadened e.g. for android, we could check adb for connected devices, and run commands on them to figure out which device is a "Samsung S4" or whatever, and then use that device.

but the driver isn't currently set up to do that. `deviceName` is also useful for cloud providers, when all you care about is getting a certain kind of device, not targeting a specific device by id.